homeassistant/components/binary_sensor/ffmpeg.py

```diff
@@ -117,7 +115,7 @@ def _service_handle_restart(service):
             _devices = DEVICES
 
         for device in _devices:
-            device.reset_ffmpeg()
+            device.restart_ffmpeg()
 
     hass.services.register(DOMAIN, SERVICE_RESTART,
```

Comment:
You are registering the service binary_sensor/restart. That is very generic. Can you rename it to binary_sensor/ffmpeg_restart